src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
```

Comment:
The modifier `.stars--4` should be placed at the root level of your SCSS file to follow BEM conventions and to keep the stylesheet organized.